### DISCUSSION TITLE
An error occurred loading the update Timeout reached for PID 42244

### DISCUSSION THREAD

**Discussion by lrhodes80** [grahampugh/erase-install#391] [Q&A]
Hi, I was wondering if anyone can help me out. 

I am using macOS Erase Install [29.1]. I have used the package and am running from Jamf Pro using the files and processing payload to run: 

/Library/Management/erase-install/erase-install.sh --os 13 --build=22F66 --update --erase --confirm 

I am trying to erase and install a copy of Ventura 13.4 on a 27" iMac 2019 on Intel. I run the policy via Jamf to the scoped iMac. It gets stuck at 8% when preparing the installer, then after about 60 minutes I get the following from terminal:

Preparing: 8.8%            An error occurred loading the update.
   [erase-install] Timeout reached for PID 42244!
2023-05-24 10:06:09  [erase-install] Reached end of script. Exit with error 42.

Any help would e greatly received. Thanks in advance.

**Comment by grahampugh** [comment]
You can try extending the timeout to 24h with the --no-timeout option.

**Reply by lrhodes80** [reply]
Hi thanks for your response. I'll try that now and report back. So from the files and processes payload I'll be running:

/Library/Management/erase-install/erase-install.sh --no-timeout --build=22F66 --erase

**Reply by lrhodes80** [reply]
It got stuck at 12% this time round. Then just hung for over an hour. Any thoughts? I really just want to erase install my labs fresh with Ventura for next Academic year. Would be good to look at the other arguments at a later date, but for now I just need the basic function.
